### PR TITLE
issue forms

### PR DESCRIPTION
**Pull request by SuperchupuDev**

### CODE REVIEW

**Review by Nyameliaaaa — APPROVED** [review]
the

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```
Comment (Nyameliaaaa) [inline .github/ISSUE_TEMPLATE/config.yml]: wont this apply to org members

**Review by Nyameliaaaa — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (SuperchupuDev) [reply .github/ISSUE_TEMPLATE/config.yml]: yeah, but it's useful so people cant randomly make empty issues

**Review by hkubus — APPROVED** [review]
I lovei t

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Nyameliaaaa) [reply .github/ISSUE_TEMPLATE/config.yml]: then we can't make issues for version projects

Comment (SuperchupuDev) [reply .github/ISSUE_TEMPLATE/config.yml]: wdym

Comment (SuperchupuDev) [reply .github/ISSUE_TEMPLATE/config.yml]: thinking about it better, ig it doesnt hurt to have it enabled, if we see its a bad idea we can just disable it later

**Review by Nyameliaaaa — APPROVED** [review]

**Pull request merged**